### PR TITLE
Remove red box around links and move appendix after references

### LINKED ISSUE #3: Appendix and References order
The default ordering in the template
1. Appendix
2. References

according to the guidelines [here](https://grad.illinois.edu/thesis/format) this should be switched.

This issue can be closed when a pull request that changes the order is created.

### PR DESCRIPTION
**Pull request by nsryan2**
## Summary of changes

This PR addresses comment @munkm made to me about removing the red boxes around hyper-linked elements. As you can see from the changes, I've added the line:
`\hypersetup{hidelinks}`
which does just that.

I also noticed that @samgdotson had an old issue about the order of sections in the template. So I moved the appendix below the references in `main.tex`. As such, this PR closes #3.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: #3 

## Checklist for Reviewers

Reviewers should use [this link](https://arfc.github.io/manual/guides/pull_requests) to get to the
Review Checklist before they begin their review.

### CODE REVIEW

**Review by ZoeRichter — APPROVED** [review]
Looks good!

File: main.tex [hunk 1]
```diff
@@ -128,14 +129,22 @@ \chapter*{Acknowledgments}
...
 
 \bibliographystyle{apalike}
 \bibliography{bibliography}
 
+\newcommand\mainmatterWithoutReset
+ {\edef\temppagenumber{\arabic{page}}%
+  \mainmatter
+  \setcounter{page}{\temppagenumber}%
+ }
+\mainmatterWithoutReset
```
Comment (yardasol) [inline main.tex:142]: very nice!

**Review by yardasol — APPROVED** [review]
Good job nathan!

**Review by ZoeRichter — APPROVED** [review]
Looks good (again)!

**Pull request merged**